backends/platform/3ds/osystem-graphics.cpp

```diff
@@ -609,16 +805,22 @@ void OSystem_3DS::setMouseCursor(const void *buf, uint w, uint h,
 
 	if (w != _cursor.w || h != _cursor.h || _cursor.format != _pfCursor) {
 		_cursor.create(w, h, _pfCursor);
-		_cursorTexture.create(w, h, _pfGameTexture);
+		_cursorTexture.create(w, h, _gfxState.gfxMode);
```

Comment:
Why not keeping the cursor texture always in RGBA so there is always alpha bits available and there is no need to draw it in software on the screen? It should not change too often nor be too expensive to convert as it is small?

Reply:
See my comment above with regards to the game and overlay formats. Same reason.